http/cookie.ts

```diff
@@ -164,8 +164,8 @@ function validateDomain(domain: string): void {
  * Parse the cookies of the Server Request
  * @param req An object which has a `headers` property
```

Comment:
```suggestion
 * @param {Headers} headers The headers instance to get cookies from.
```

Need to take care to keep the jsdocs in line with changes 🙂

Reply:
By way of "improve what you touch", consider also adding the `@return {Object} …` jsdoc for these methods as well to add clarity to what is returned for users of the Deno docs.